scripts/prepare-main-after-release.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Removes all changesets from provided release branch and bumps the version
+
+set -e
+
+git diff --name-only origin/"$1" .changeset/ | xargs git rm
```

Comment:
Tell `git rm` that you are passing only files:

```suggestion
git diff --name-only origin/"$1" .changeset/ | xargs git rm --
```

c.f. manual git-rm(1) for details